src/test/clojure/jarl/builtins/utils_test.clj

```diff
@@ -2,24 +2,24 @@
   (:require [clojure.test :refer [deftest is testing]]
             [jarl.exceptions :as errors]
             [jarl.builtins.utils :refer [typed-seq]])
-  (:import (se.fylling.jarl BuiltinException)))
+  (:import (se.fylling.jarl BuiltinException TypeException)))
 
 (deftest builtin-ex-test
   (testing "formatting"
     (let [ex (errors/builtin-ex "my message #%d caused by %s" 1 "incompetence")]
       (is (instance? BuiltinException ex))
-      (is (= (.getMessage ^Throwable ex) "my message #1 caused by incompetence"))))
+      (is (= (.getMessage ^Throwable ex) "eval_builtin_error: my message #1 caused by incompetence"))))
```

Comment:
This string-match fails. Could it be that `^Throwable` causes a down-cast, so that `getMessage` isn't handed by `JarlException`?